packages/options/options.pony

```diff
@@ -101,6 +101,14 @@ class Options is Iterator[(ParsedOption | ParseError | None)]
       _arguments.push(arg.clone())
     end
 
+  new from_array(args: Array[String val] box, fatal: Bool = true) =>
```

Comment:
I don't know how behave `embed` fields, I know to little about them, so I just assumed that cloning is required with them.
And so, since Env.args is already an `Array[String val] val`, the existing code is already doing extra cloning ?